### PR TITLE
[Spinner] Docs , SPEC.md & Props Fix

### PR DESCRIPTION
**Pull request by rohanprasadofficial**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [x] android

### Description of changes
This targets to update the SPEC.md files for Spinner. 
It also consist of props fixing and removal of non required props. 

### Verification

(how the change was tested, including both manual and automated tests)

| Current Version                                      | 
|----------------------------------------------|
| Screenshot_1675846122 | 

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [x] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Review by lenahong — APPROVED** [review]

File: packages/experimental/Spinner/SPEC.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+## Sample Code
+
+### Basic Mobile Spinner
+
+```tsx
+<>
+  <Spinner /> // This is a spinner with default size as 'medium'
+  <Spinner size="x-small" /> // Spinner with a defined size
+  <Spinner status={status} /> // status property depicts it is animating or not
```
Comment (ankraj12) [inline packages/experimental/Spinner/SPEC.md:22]: Does this naming align with web api's for spinner? If not, will a name like "`isRotating`" be more readable?

Comment (ankraj12) [inline packages/experimental/Spinner/SPEC.md:22]: nit: please follow same casing across comments

**Review by ankraj12 — APPROVED** [review]
Please review the minor comments and feel free to close post it.

File: packages/experimental/Spinner/SPEC.md (hunk 1 above)
Comment (rohanprasadofficial) [reply packages/experimental/Spinner/SPEC.md:22]: status is enum which has value as "active" and "unactive" which depicts the spinning which aligns with web api. 
I can change the comments though to "depicts it is rotating or not"

Comment (rohanprasadofficial) [reply packages/experimental/Spinner/SPEC.md]: added ' ' to the property.

**Pull request merged**